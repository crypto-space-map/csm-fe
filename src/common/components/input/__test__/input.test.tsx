import { render } from '@testing-library/react';

import { Input } from '..';

describe('<Input />', () => {
  it('should match snapshot', () => {
    const { container } = render(<Input value="snapshot" errorMessage="This is ERROR" />);
    expect(container).toMatchSnapshot();
  });
  it('should be defined', () => {
    const { container } = render(<Input value="snapshot" errorMessage="This is ERROR" />);
    expect(container).toBeDefined();
  });
});
