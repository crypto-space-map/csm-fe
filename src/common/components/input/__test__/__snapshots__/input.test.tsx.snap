// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Input /> should match snapshot 1`] = `
<div>
  <div
    class="css-13v5l7y"
  >
    <span
      class="css-q66kqr"
    />
    <div
      class="MuiFormControl-root MuiTextField-root css-hjkj6l-MuiFormControl-root-MuiTextField-root"
      errormessage="This is ERROR"
    >
      
      <div
        class="MuiInput-root MuiInputBase-root MuiInputBase-colorPrimary MuiInputBase-formControl css-1aa5qj0-MuiInputBase-root-MuiInput-root"
      >
        <input
          aria-invalid="false"
          class="MuiInput-input MuiInputBase-input css-1x51dt5-MuiInputBase-input-MuiInput-input"
          type="text"
          value="snapshot"
        />
      </div>
    </div>
  </div>
</div>
`;
